setup.py

```diff
@@ -55,7 +55,7 @@
         ]
     },
     python_requires=">=3.6.0",
-    install_requires=["torch>=1.4.0", "pyyaml", "numpy>=1.17"],
+    install_requires=["numpy>=1.17", "pyyaml", "torch>=1.4.0", "packaging>=20.0"],
```

Comment:
If we're alphabetically sorting, let's really do it ;-)

Reply:
Fixed, heh